source/search.haml

```diff
@@ -8,9 +8,14 @@ noindex: true
 .container
   .row
     - standard_width_div do
+      / Input group is a bootstrap element we use for adding a spinner addon.
```

Comment:
A link to the bootstrap docs about this might be helpful.

Also, we're using fontawesome for the spinner, a note/link about that would be prudent.